testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/management/cli/UndeployTestCase.java

```diff
@@ -114,10 +134,22 @@ public void testLegacyUndeployWithAllServerGroups() throws Exception {
         checkExist(result, cliTestApp1War.getName(), ENABLED, ctx);
 
         // From serverGroup1 only, still referenced from sg2. Must keep-content
-        ctx.handle("undeploy --server-groups=" + sgOne + ' ' + cliTestApp1War.getName());
+        try {
+            ctx.handle("undeploy --server-groups=" + sgOne + ' ' + cliTestApp1War.getName());
+        } catch (CommandLineException ex) {
+            // Check error message
+            assertThat("Error message doesn't contains expected message information!",
+                    ex.getMessage(),
+                    allOf(containsString("Cannot undeploy"),
+                            containsString(cliTestApp1War.getName()),
+                            containsString(" as it is still deployed in "),
+                            containsString(sgTwo),
+                            containsString("Please specify all relevant server groups.")));
+            // Verification wrong command execution fail - success
+        }
 
         result = legacyDeploymentInfo(cli, sgOne);
-        checkExist(result, cliTestApp1War.getName(), ADDED, ctx);
+        checkExist(result, cliTestApp1War.getName(), ENABLED, ctx);
```

Comment:
Does this change backward compatibility for previous server version? I'm not sure because these tests doesn't exist earlier.

Reply:
see the comment above